uint16 gpioId
uint8 edgeChangeType
---
uint32 callbackId
bool result